main.css

```diff
@@ -28,17 +28,23 @@ body {
   color: #4d4d4d;
 }
 
+a {
+  transition: all .2s;
+}
+
 a:link {
   color: #fff;
   text-decoration: none;
 }
 
 a:visited {
-  color: hsl(48, 89%, 80%);
+  color: hsl(48, 89%, 87%);
 }
 
 a:hover {
-  text-decoration: underline;
+  /*text-decoration: underline;*/
+  text-shadow: 2px 2px 5px #222;
+
```

Comment:
I'd recommend removing the commented out line here. Git allows you to see changes over time, so there's rarely a need to leave commented code.

Also minor thing, but you have an extra blank line here that can be removed.